django_socio_grpc/proto_serializers.py

```diff
@@ -71,20 +75,22 @@ def data_to_message(self, data):
         return parse_dict(data, self.Meta.proto_class())
 
 
-class ListProtoSerializer(BaseProtoSerializer, ListSerializer):
+class ListProtoSerializer(ListSerializer, BaseProtoSerializer):
     def message_to_data(self, message):
         """
         List of protobuf messages -> List of dicts of python primitive datatypes.
         """
-        if not isinstance(message, list):
-            error_message = self.error_messages["not_a_list"].format(
-                input_type=type(message).__name__
+
+        real_message = getattr(message, self.message_list_attr)
```

Comment:
check if message_list_attr is not None and raise an error is fo.

add a default return value if message doesn't have self.message_list_attr attribute